### PR TITLE
fixed visiting project details for a admin user.

### PR DESCRIPTION
**Pull request by wknet123**
fixed visiting project details issues for a admin user.

### CODE REVIEW

File: controllers/itemdetail.go [hunk 1]
```diff
@@ -64,6 +64,8 @@ func (idc *ItemDetailController) Get() {
 
 	if sessionUserID != nil {
 
+		userID := sessionUserID.(int)
+
 		idc.Data["Username"] = idc.GetSession("username")
 		idc.Data["UserId"] = sessionUserID.(int)
```
Comment (reasonerjt) [inline controllers/itemdetail.go]: use the var userID

File: controllers/itemdetail.go [hunk 2]
```diff
@@ -73,12 +75,20 @@ func (idc *ItemDetailController) Get() {
...
+		}
+
+		if !isAdmin && (project.Public == 0 && len(roleList) == 0) {
 			idc.Redirect("/registry/project", http.StatusFound)
 			return
 		}
 
-		if len(roleList) > 0 {
+		if isAdmin {
+			idc.Data["RoleId"] = models.SYSADMIN
```
Comment (reasonerjt) [inline controllers/itemdetail.go]: Please check how idc.Data["RoleId"] is used.

**Comment by reasonerjt** [comment]
LGTM

**Comment by ywk253100** [comment]
LGTM

**Pull request merged**